File: app/src/main/java/game/ivan/ashancalculator/tags/view/TagsView.java
package game.ivan.ashancalculator.tags.view;

import com.hannesdorfmann.mosby.mvp.MvpView;

import java.util.List;

import game.ivan.ashancalculator.database.models.Tags;
import io.reactivex.Observable;

/**
 * Created by ivan on 20.12.16.
 */

public interface TagsView extends MvpView {

    void refreshList(Observable<List<Tags>> list);

    public void showEditDialog(Tags tag);

    void showCreateDialog();
}

File: app/src/main/java/game/ivan/ashancalculator/tags/view/TagsView.kt
package game.ivan.ashancalculator.tags.view

import com.hannesdorfmann.mosby.mvp.MvpView

import game.ivan.ashancalculator.database.models.Tags

/**
 * Created by ivan on 20.12.16.
 */

interface TagsView : MvpView {

    fun refreshList(list: List<Tags>)

    fun showEditDialog(tag: Tags)

    fun showCreateDialog()
}
